airgun/entities/job_invocation.py

```diff
@@ -85,6 +87,22 @@ def read_hostgroups(self):
         )
         return view.target_hosts_and_inputs.targets.items
 
+    def dynflow_output(self, entity_name, host_name):
```

Comment:
```suggestion
    def read_dynflow_output(self, entity_name, host_name):
```
again I would suggest naming the function in a way that better describes its behavior